src/flamegpu/io/Telemetry.cpp

```diff
@@ -0,0 +1,167 @@
+#include "flamegpu/io/Telemetry.h"
+
+#include <cstdlib>
+#include <cerrno>
+#include <algorithm>
+#include <memory>
+#include <array>
+#include <iostream>
+#include <cstdio>
+#include <cctype>
+
+
+#include "flamegpu/version.h"
+#include "flamegpu/exception/FLAMEGPUException.h"
+#include "flamegpu/util/Environment.h"
+
+
+
+namespace flamegpu {
+namespace io {
+namespace Telemetry {
+
+const char TELEMETRY_ENDPOINT[] = "https://nom.telemetrydeck.com/v1/";
```

Comment:
These should be in an anonymous namespace.

Reply:
`TELEMETRY_ENDPOINT` is currently accessed in other files (used to output to users. Probably provide a method to get the endpoint instead?